torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -248,6 +255,103 @@ def _should_disable_saved_tensors_hooks():
     return False
 
 
+# Custom ops that intentionally mutate inputs (e.g., in-place fusion ops)
+# and should skip the aliasing constraint check
+_CUSTOM_OP_ALIASING_ALLOWLIST = [
+    torch.ops.mkldnn._convolution_pointwise_.binary,
+]
+
+
+def _is_ci() -> bool:
```

Comment:
nit: instead of checking for ci directly in the AOTAutograd code, we should:

(1) have a config in AOT that determines if we warn or error

(2) have the CI scripts set that config to error

Reply:
I guess I'm wondering if whatever error you hit is a real silent correctness case we should fix